doc/api/buffer.md

```diff
@@ -872,15 +877,17 @@ console.log(buf.toString('ascii'));
 added: v0.11.13
 -->
 
-* `target` {Buffer}
+* `target` {Buffer} A `Buffer` to compare to
 * `targetStart` {Integer} The offset within `target` at which to begin
-  comparison. default = `0`.
-* `targetEnd` {Integer} The offset with `target` at which to end comparison.
-  Ignored when `targetStart` is `undefined`. default = `target.byteLength`.
+  comparison. **Default:** `0`
+* `targetEnd` {Integer} The offset with `target` at which to end comparison
+  (not inclusive). Ignored when `targetStart` is `undefined`.
```

Comment:
Should the docs indicate `unsigned`? Since we're putting `{Integer}` instead of `{Number}`, seems having it a little more defined makes sense. Maybe `{Uint32}`?

Also, I'm honestly not sure what "not inclusive" is trying to tell me here.